docs/setup_ovos_services.md

```diff
@@ -173,6 +173,9 @@ StartLimitBurst=4
 WantedBy=ovos.service
 ```
 
+### Setup the OpenVoiceOS service on Raspbian
+- Check [Services in Rasbian](./docs/setup_ovos_services.raspian.md)
```

Comment:
This is in the process of being changed to just an install shell script.  Not sure it is applicable anymore.